src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -273,9 +273,14 @@ $fullscreen-modal-padding: 64px;
   .k-modal-fullscreen-header-description {
     color: var(--KModalFullscreenHeaderColor, var(--black-500, color(black-500)));
     display: flex;
+    flex-direction: column;
     font-size: var(--KModalFullscreenHeaderSize, 20px);
     font-weight: var(--KModalFullscreenHeaderWeight, 600);
     justify-content: space-between;
+
+    @media only screen and (min-width: ($viewport-md)) {
```

Comment:
nit: remove `screen` -- since we're never worried about the `print` layout, etc. you can just remove it

Reply:
Done